media-sound/audacity/audacity-3.4.2.ebuild

```diff
@@ -97,7 +97,10 @@ RDEPEND="dev-db/sqlite:3
 	)
 	mpg123? ( media-sound/mpg123 )
 	ogg? ( media-libs/libogg )
-	opus? ( media-libs/opus )
+	opus? (
+		media-libs/opus
+		media-libs/opusfile
+	)
```

Comment:
> Because it needs a bump, I would consider solving `media-sound/mpg123-base` in this PR (albeit it can be as a separate commit), else that would mean another bump.
> 
> And, I spotted another one: looks like `dev-libs/rapidjson` is headers only. I think meaning it would be a DEPEND only, and not trigger a rebuild on slot change.

I've addressed these imperfections in 05d838a28b1c2516741f7ff25b2c4e89984e76e8.